src/imf_reader/sdr/clear_cache.py

```diff
@@ -0,0 +1,39 @@
+from imf_reader.sdr.read_announcements import (
+    get_holdings_and_allocations_data,
+    get_latest_date,
+)
+from imf_reader.sdr.read_exchange_rate import fetch_exchange_rates
+from imf_reader.sdr.read_interest_rate import fetch_interest_rates
+from imf_reader.config import logger
+
+
+def clear_cache():
+    """Clear the cache for all lru_cache-decorated functions."""
+
+    cleared_caches = 0
+
+    # read_announcements
+    if (
+        get_holdings_and_allocations_data.cache_info().currsize > 0
+        or get_latest_date.cache_info().currsize > 0
+    ):
+        get_holdings_and_allocations_data.cache_clear()
+        get_latest_date.cache_clear()
```

Comment:
`cache_clear` should function normally even if there is no cached data, so the condition check and counter might be redundant